diff --git a/CMakeLists.txt b/CMakeLists.txt
index 562c70e3..70cb08a9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -185,7 +185,7 @@ if (${BUILD_GEOCODER} STREQUAL "ON")
   )

   add_custom_command (
-    COMMAND generate_geocoding_data "${GEOCODING_DIR}"
+    COMMAND $ENV{NATIVE_TOOLS_DIR}/generate_geocoding_data "${GEOCODING_DIR}"
       "${GEOCODING_DATA_OUTPUT}"

     OUTPUT ${GEOCODING_DATA_OUTPUT}
@@ -462,7 +462,7 @@ if (${BUILD_GEOCODER} STREQUAL "ON")
   )
 
   add_custom_command (
-    COMMAND generate_geocoding_data "${GEOCODING_TEST_DIR}"
+    COMMAND $ENV{NATIVE_TOOLS_DIR}/generate_geocoding_data "${GEOCODING_TEST_DIR}"
       "${GEOCODING_TEST_DATA_OUTPUT}" "_test"
 
     OUTPUT ${GEOCODING_TEST_DATA_OUTPUT}
